src/js/utils/portal.js

```diff
@@ -1,4 +1,5 @@
-import css from 'css';
+import css from 'postcss'; // eslint-disable-line max-len, import/no-extraneous-dependencies
+import prettier from 'prettier'; // eslint-disable-line max-len, import/no-extraneous-dependencies
```

Comment:
why do we need all the linter comments?

Reply:
The `eslint` rules correctly disallow devDependecies from being imported in the regular source code. However because in `utils` all types of files are thrown in together, some of these are false positives. It is possible to change the global policy or override the rule itself in `.eslintrc` but I thought that was too heavy-handed and could lead to regressions. So I chose to waive the violations per line, however the waiver comments make the line go over the max-len limit so also waived that inline.

Options are:
- Leave as is.
- Waive the block, i.e. disable then enable with the comments above or below.
- Add an additional file to the override block in `.eslintrc` where more rules are also modified or create a new override block just for this file.
- Specify the lint rule globally "specifically" for Grommet throwing away the definition in `eslint-config-airbnb`. If this is what you'd prefer it is better to file it as a new issue. Example:
```json
    'import/no-extraneous-dependencies': ['error', {
      devDependencies: [
        '**/__tests__/**',
        '*.stories.js',
        '**/stories/**'
        '**/portal.js',
        '**/webpack.config.*.js'
      ],
      optionalDependencies: false,
    }],
```

Reply:
I'm ok with leaving it as is. thanks for clarifying.